src/screens/TrackTileSettingsScreen.test.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { View } from 'react-native';
+import { fireEvent, render } from '@testing-library/react-native';
+import SettingsScreen from './TrackTileSettingsScreen';
+import { useTrackers } from '../components/TrackTile/hooks/useTrackers';
+import { TrackTileServiceProvider } from '../components/TrackTile/main';
+
+jest.mock('../components/HeaderButton', () => ({
+  HeaderButton: jest.requireActual('react-native').Button,
+}));
+jest.mock('../components/TrackTile/hooks/useTrackers', () => ({
+  useTrackers: jest.fn(() => ({})),
+}));
+jest.mock('i18next', () => ({
```

Comment:
I'm assuming you mocked this because you are relying on some interpolation that is not available because the set defaultValues should cover most cases. In the few cases it is not covered you could manually add those entries here: https://github.com/lifeomic/react-native-sdk/blob/f1c6bbf1b98132af90732ea650bf106002f8e158/jest.setup.ts#L105 

This, of course, does not really add to the overall value of the test so definitely consider this a nit.

Reply:
I'm actually not sure why I mocked that. Removed!